backpack/__init__.py

```diff
@@ -66,6 +74,15 @@ def __enter__(self):
         CTX.set_extension_hook(self.extension_hook)
 
     def __exit__(self, type, value, traceback):
+        """Leave backpack environment.
+
+        Args:
+            type: .
+            value: .
+            traceback: .
+        """
```

Comment:
Re types and argument descriptions: I found [this answer](https://stackoverflow.com/a/58808179) and [this example code](https://github.com/python/typeshed/blob/095464874accef1cc5303f325ccabe88756bcd8b/stdlib/2and3/contextlib.pyi#L67-L69).

Reply:
Awesome!